distributed-testing/build.gradle

```diff
@@ -0,0 +1,63 @@
+apply plugin: 'groovy'
+apply plugin: 'java-gradle-plugin'
+
+repositories {
+    mavenLocal()
+    mavenCentral()
+    jcenter()
+}
+
+gradlePlugin {
+    plugins {
+        distributedTestingPlugin {
+            id = 'net.corda.plugins.distributed-testing'
+            implementationClass = 'net.corda.plugins.DistributedTesting'
+        }
+    }
+    automatedPublishing = false
+}
+
+
+pluginBundle {
+    website = project.vcsUrl
+    vcsUrl = project.vcsUrl
+    description = 'Corda Distributed Testing Plugin'
+    tags = ['corda']
+
+    plugins {
+        distributedTestingPlugin {
+            displayName = 'Corda Distributed Testing Plugin'
+        }
+    }
+}
+
+dependencies {
+    compile gradleApi()
+    compile "io.fabric8:kubernetes-client:4.4.1"
+    compile 'org.apache.commons:commons-compress:1.19'
+    compile 'org.apache.commons:commons-lang3:3.9'
+    compile 'commons-codec:commons-codec:1.13'
+    compile "io.github.classgraph:classgraph:4.8.41"
+    compile "com.bmuschko:gradle-docker-plugin:5.0.0"
+    compile 'org.apache.commons:commons-csv:1.1'
+    compile group: 'org.jetbrains', name: 'annotations', version: '13.0'
+    testCompile "junit:junit:4.12"
```

Comment:
You (presumably) don't want `junit:junit:4.12` if you're using the Jupiter engine. The other Gradle plugins are all using Jupiter.